### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by robhyrk**
- [x] add block number
- [x] fix whitespace issue upon open
- [x] fix logo and hamburger menu colors
- [x] fix menu, remove scroll

### CODE REVIEW

**Comment by robhyrk** [comment]
I've removed the top-bar component and placed all menu related components to the menu folder for simplicity. This also helped with keeping the side menu aligned with the logo's relative position. 

I also removed all instances of mob-x stores and used nextJS's built in routing hook for the menu logic

**Comment by robhyrk** [comment]
Also, I tried removing some code related to left/right drawers from store but it was causing build errors since it relied on other components. I was going to remove them but I'm not sure if it's ready to do so; might be worth revisiting and cleaning this up at some point.

File: components/menu/Navigation.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React, { FC } from "react";
+import { MenuItem, MenuItemMobile } from "./MenuItem";
+import { useRouter } from "next/router";
+
+const Navigation: FC<{
+  navigation: {};
```
Comment (Robiquet) [inline components/menu/Navigation.tsx:6]: Is there a better type for this?

File: layouts/DefaultLayout.tsx [hunk 2]
```diff
@@ -52,13 +45,12 @@ const DefaultLayout: FC<PropsWithChildren> = observer(({ children }) => {
             }
           `}
         </style>
-        {process.env.NEXT_PUBLIC_MIGRATION_IN_PROGRESS !== "true" && <Menu />}
```
Comment (Robiquet) [inline layouts/DefaultLayout.tsx:55]: Is this correct? How does this app look when `process.env.NEXT_PUBLIC_MIGRATION_IN_PROGRESS !== "true"` evaluates to `true`?

Comment (robhyrk) [reply layouts/DefaultLayout.tsx:55]: That was a mistake from a merge. Have since removed it.

File: components/menu/Navigation.tsx (hunk 1 above)
Comment (robhyrk) [reply components/menu/Navigation.tsx:6]: Oops, yeah that was meant as a placeholder.  I have made an update in the latest push.

**Review by Robiquet — APPROVED** [review]

**Pull request merged**